iosApp/iosApp/Pages/NearbyTransit/NoNearbyStopsView.swift

```diff
@@ -0,0 +1,76 @@
+//
+//  NoNearbyStopsView.swift
+//  iosApp
+//
+//  Created by Horn, Melody on 2024-11-07.
+//  Copyright © 2024 MBTA. All rights reserved.
+//
+
+import SwiftUI
+
+struct NoNearbyStopsView: View {
```

Comment:
👍 I like the way you split this out

Reply:
I rely pretty heavily on Xcode previews to steer my initial implementation, and mocking enough data in the NearbyTransitView itself to be able to see this state in previews was really difficult.